WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -288,9 +309,13 @@ private extension PaymentCaptureOrchestrator {
 
 private extension PaymentCaptureOrchestrator {
     enum Constants {
-        /// Minimum order amount in USD:
+        /// Minimum order amount in USD or CAD:
         /// https://stripe.com/docs/currencies#minimum-and-maximum-charge-amounts
         static let minimumAmount = NSDecimalNumber(string: "0.5")
+
+        static let canadaFlatFee = NSDecimalNumber(string: "0.25")
+
+        static let canadaPercentageFee = NSDecimalNumber(string: "0.026")
```

Comment:
>Just wanted to confirm that we're using these numbers while the [WCPay doc for Canada](https://woocommerce.com/document/payments/faq/fees/#section-4) shows different values?

The numbers I used were the numbers provided by @kalessil that WCPay use, but we're getting them confirmed with Stripe. I've left them matching WCPay's implementation until we hear differently.

> Do we currently handle these two cases on the server-side, and similarly on the client-side?

We don't handle them at all on Interac payments, but Interac cards are all Canadian-issued, so these scenarios don't apply. 

For all other payments they would/should be handled on the server side... but they're likely to be very rare anyway because the apps don't really support multi-currency. I think it might be possible to reproduce it with an order placed on the web for a multi-currency site and then paid for as CoD.